storagenode/storagenodedb/infodb.go

```diff
@@ -269,6 +271,45 @@ func (db *InfoDB) Migration() *migrate.Migration {
 					`CREATE INDEX idx_pieceinfo_expiration ON pieceinfo(piece_expiration) WHERE piece_expiration IS NOT NULL`,
 				},
 			},
+			{
+				Description: "Clear Tables from Alpha data",
+				Version:     11,
+				Action: migrate.SQL{
+					`DELETE FROM pieceinfo`,
+					`DELETE FROM used_serial`,
+					`DELETE FROM order_archive`,
+				},
+			},
+			{
+				Description: "Free Storagenodes from trash data",
+				Version:     12,
+				Action: migrate.Func(func(log *zap.Logger, mgdb migrate.DB, tx *sql.Tx) error {
+					// When using inmemory DB, skip deletion process
+					if db.location == "" {
+						return nil
+					}
+
+					err := os.RemoveAll(filepath.Join(filepath.Dir(db.location), "blob/ukfu6bhbboxilvt7jrwlqk7y2tapb5d2r2tsmj2sjxvw5qaaaaaa")) // us-central1
+					if err != nil {
+						log.Sugar().Debug(err)
+					}
+					err = os.RemoveAll(filepath.Join(filepath.Dir(db.location), "blob/v4weeab67sbgvnbwd5z7tweqsqqun7qox2agpbxy44mqqaaaaaaa")) // europe-west1
+					if err != nil {
+						log.Sugar().Debug(err)
+					}
+					err = os.RemoveAll(filepath.Join(filepath.Dir(db.location), "blob/qstuylguhrn2ozjv4h2c6xpxykd622gtgurhql2k7k75wqaaaaaa")) // asia-east1
+					if err != nil {
+						log.Sugar().Debug(err)
+					}
+					err = os.RemoveAll(filepath.Join(filepath.Dir(db.location), "blob/abforhuxbzyd35blusvrifvdwmfx4hmocsva4vmpp3rgqaaaaaaa")) // "tothemoon (stefan)"
+					if err != nil {
+						log.Sugar().Debug(err)
+					}
+					// If a folder is not found, the above command is returning an err.
```

Comment:
According the GoDoc https://golang.org/pkg/os/#RemoveAll the function will return a `nil` error if the folder path does not exists, otherwise it will return an error.

Not sure if we want to skip all the errors.

Reply:
Yes, from what i heard, we dont want to block the SN from starting if it fails to delete data.
As we moved into a new blobs directory, we can just delete it later on manually if users have issues with that.

Reply:
OK, then remove that comment because it's incorrect and cause confusion